data-reconciliation-app/docs/adr/01-data-ingest.md

```diff
@@ -2,126 +2,150 @@
 
 ## Status
 
-Proposed
+Approved
 
 ## Context
 
-We need to build an API that ingests member data into our system. Once ingested, we also need to store this data in the K-V store.
+We need to build an API that ingests data into our system. Once ingested, we also need to store this data in the K-V store so we can reconcile and report out on the data.
 
 Gathered requirements:
 
-- Members will decide upon a csv schema ahead of data ingest
-- Members will upload their data via csv, with headers
-  - csv will have 2 columns:
-    - one unique identifier (string)
-    - one attribute associated with the unique identifier (string)
+- Members of the network will decide on a data schema ahead of ingest.
+- Members and Users of the network can ingest data via the API. Data will be ingested via JSON which will include:
+  - one unique identifier (string)
+  - one attribute associated with the unique identifier (string)
 - There is likely going to be a UI on top of our ingest API...so we will treat our app like a backend system. Therefore, we will accept JSON into our ingest API.
 
 ## Decision
 
-## Model for K-V store
+## API
 
-```
-//psuedo code for model based off forum app
+### Endpoint
 
-// this is unique for each user
-type User = string;
+- Description
+  - Ingest member or user data into our application and add data to K-V store
+- Path
+  - /ingest
+- HTTP Method
+  - POST
+- URL Params
+  - N/A
+- Headers
+  - content-type: application/json
+- Request will be a `ccfapp.Request Object`. We will interrogate the `ccfapp.Request Object` to extract the user or member and [authenticate them via certficates](#security). We will read the request body as a JSON.
+- API Status Codes
+  - OK
+    - Status: 200
+    - Status Text: "Votes have been successfully recorded"
```

Comment:
This will be `Data has been ingested` message then?

Reply:
Good catch, thanks. Will update